packages/babel-plugin-transform-computed-properties/src/index.ts

```diff
@@ -26,6 +27,44 @@ export default declare((api, options: Options) => {
     ? pushComputedPropsLoose
     : pushComputedPropsSpec;
 
+  function buildDefineAccessor(
+    state: PluginPass,
+    type: "get" | "set",
+    obj: t.Expression,
+    key: t.Expression,
+    fn: t.Expression,
+  ) {
+    let helper: t.Identifier;
+    if (state.availableHelper("defineAccessor")) {
+      helper = state.addHelper("defineAccessor");
+    } else {
+      // Fallback for @babel/helpers <= 7.20.6, manually add helper function
+      const file = state.file;
+      helper = file.declarations["defineAccessor"];
+      if (!helper) {
+        helper = file.declarations["defineAccessor"] =
+          file.scope.generateUidIdentifier("defineAccessor");
+
+        const helperDeclaration = template.statement.ast`
+          function ${helper}(type, obj, key, fn) {
+            var desc = { configurable: true, enumerable: true };
+            desc[type] = fn;
+            return Object.defineProperty(obj, key, desc);
+          }      
+        `;
+        const helperPath = file.path.unshiftContainer(
+          "body",
+          helperDeclaration,
+        )[0];
+        // TODO: NodePath#unshiftContainer should automatically register new bindings.
+        file.scope.registerDeclaration(helperPath);
+      }
+      helper = t.cloneNode(helper);
+    }
+
+    return t.callExpression(helper, [t.stringLiteral(type), obj, key, fn]);
+  }
```

Comment:
Now the helper function is directly added to the file scope if it is not available. For example:
```js
var obj = {
  get ["x" + foo]() { return "heh"; }
};
```
The output would be:
```diff
- var obj = babelHelpers.defineAccessor("get", {}, "x" + foo, function () {
+ function _defineAccessor(type, obj, key, fn) {
+   var desc = {
+     configurable: true,
+     enumerable: true
+   };
+   desc[type] = fn;
+   return Object.defineProperty(obj, key, desc);
+ }
+ var obj = _defineAccessor("get", {}, "x" + foo, function () {
    return "heh";
  });
```